project_event/views/project_task_view.xml

```diff
@@ -165,13 +167,16 @@
                             <field name="project_id" invisible="1"/>
                             <field name="responsible_id" attrs="{'readonly': [('task_state', 'in', ['requested','accepted','done'])]}" required="1"/>
                             <field name="partner_id" attrs="{'readonly': [('task_state', 'in', ['requested','accepted','done'])]}"/>
+                            <field name="client_type"/>
+                            <field name="sector_id"/>
```

Comment:
@jananjoy the sector field should be visible only if the client_type is not empty

Reply:
@rimbendhaou Done